src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,100 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+import java.util.Objects;
```

Comment:
Don't use Objects, Arrays, or any other util class. In this case, `Objects` is used, which is not allowed as per the checklist.